apps/web/components/user-feedback/dynamic-feedback-table.tsx

```diff
@@ -47,7 +50,9 @@ function DynamicFeedbackTable({
             <SkeletonFeedbackItem key={i} />
           ))
         ) : error ? (
-          <NoFeedbackMessage type="error" onRetry={onRetry} />
+          <NoFeedbackMessage
```

Comment:
Usually, it is better to avoid the ternary operator in JSX because it becomes unreadable.

You can use such constructions:

```tsx
{!isLoading && error && (<Tag />)
{isLoading && !data && (<LoadingState />)}
{!isLoading && data && (<Data />)}
```

Reply:
This is much easier to read